web/src/hooks/useWindowDimensions.ts

```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect } from 'react'
 
-const VIEWPORT_SMALL_THRESHOLD = 750
+const VIEWPORT_SMALL_THRESHOLD = 768
```

Comment:
Why is this such a weird number?

Reply:
this is a standard viewport width used in many frameworks. 750 is weird ;) f.e. here under medium
https://getbootstrap.com/docs/4.0/layout/grid/#grid-options